recipe/meta.yaml

```diff
@@ -12,14 +12,11 @@ source:
     - no-console.patch
 
 build:
-  number: 2
-  features:
-    - vc9  # [win and py27]
-    - vc14  # [win and py>=35]
-  skip: true  # [win and py36]
+  number: 4
```

Comment:
No need to bump the number for PRs that never merged; 3 is fine. :)

Reply:
Revision 3 was published as I merged accidentally.